components/popover/popover-mixin.js

```diff
@@ -0,0 +1,99 @@
+import '../colors/colors.js';
+import { css, html } from 'lit';
+
+const isSupported = ('popover' in HTMLElement.prototype);
+
+// eslint-disable-next-line no-console
+console.log('Popover', isSupported);
+
+export const PopoverMixin = superclass => class extends superclass {
+
+	static get properties() {
+		return {
+			/**
+			 * Whether the popover is open or not
+			 * @type {boolean}
+			 */
+			opened: { type: Boolean, reflect: true },
```

Comment:
I could be convinced to go with either `opened` or `showing`. The native API uses `show/hidePopover` for the methods, but then uses `:popover-open` for the CSS pseudo-class. Our dialog and dropdown use `opened`, while our tooltips use `showing`. 🤷‍♂️

Reply:
Oh interesting - should we get these consistent between dropdown and tooltip? If we did want to do that, `d2l-tooltip` + `showing` has far less usages (but overall it's probably not super necessary)

I'm fine with `opened` here.

Reply:
It's a bikeshed moment. 😄  I'm good either way, and we can easily switch with subsequent changes.